app/com/linkedin/drelephant/spark/fetchers/SparkRestClient.scala

```diff
@@ -275,8 +279,9 @@ object SparkRestClient {
   val API_V1_MOUNT_PATH = "api/v1"
   val IN_PROGRESS = ".inprogress"
   val DEFAULT_TIMEOUT = Duration(5, SECONDS)
-  val CONNECTION_TIMEOUT = 5000
-  val READ_TIMEOUT = 5000
+  val TEN_SECONDS_TIMEOUT = Duration(10, SECONDS)
```

Comment:
Please change the variable name (TEN_SECONDS_TIMEOUT) to something better ,so that it can be general

Reply:
Would MAX_PERMITTED_TIMEOUT work?

Reply:
Yes better